### ISSUE TITLE
Crystal - Shadow Sorcerer Cracktro

### ISSUE THREAD

**Issue by chris70c** [dirkwhoffmann/vAmiga#575]
Last right part of the scroller seems to be weird/missing.
![vamiga](https://user-images.githubusercontent.com/3858768/138730803-2312a2e3-a3fa-4415-85b4-5389702feb73.png)
![winuae](https://user-images.githubusercontent.com/3858768/138730965-79bba636-b7ff-4d21-9931-5a01dc9cf91c.png)
[Crystal-CrShadowSorcerer.zip](https://github.com/dirkwhoffmann/vAmiga/files/7411891/Crystal-CrShadowSorcerer.zip)

**Comment by dirkwhoffmann** [comment]
I think this is related to the location of the HBLANK area. Here is the whole texture:

<img width="1024" alt="Bildschirmfoto 2021-11-06 um 10 55 24" src="https://user-images.githubusercontent.com/12561945/140605627-a4fbf678-8fc8-42a0-aef9-11b0cc421ec1.png">

The red lines mark the refresh cycles aka DMA cycles $E2, $01, $03, $05.

On a real Amiga, the image on the Monitor is shifted by an offset which means that DMA cycle 0 does *not* mark the beginning of a new line. Its position on the screen is part of the previous line. The correct offset value depends on the actual HBLANK area. I remember when I started to implement vAmiga I had problems finding out at which DMA cycle HBLANK starts and at which DMA cycle HBLANK ends. E.g., "Amiga Intern" which I used a lot, mentioned different values than other sources. 

Next step: Find proper information about the HBLANK area.
